.eslintrc.js

```diff
@@ -2,8 +2,12 @@ module.exports = {
   "extends": [
     "react-app",
     "prettier/@typescript-eslint",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
```

Comment:
I think `plugin:prettier/recommended` should always be used as the last item in the array to make sure all rules that conflict with Prettier are overwritten

Reply:
i see ok, i can actually remove the one i added, i was trying to get rid of the error i was getting